src/main/java/org/madhuri/app/message/model/Message.java

```diff
@@ -1,91 +1,121 @@
 package org.madhuri.app.message.model;
 
 import java.util.Date;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
 import jakarta.json.bind.annotation.JsonbProperty;
-import jakarta.json.bind.annotation.JsonbDateFormat; 
-
-
-public class Message{
-
-    @JsonbProperty("id")
-    private long id;
-
-    @JsonbProperty("messageContent")
-    private String messageContent;
-
-    @JsonbProperty("created")
-    @JsonbDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
-    private Date created;
-
-    @JsonbProperty("username")
-    private String username;
-    
-
-    @JsonbProperty("recipient")
-    private String recipient;
-
-
-    public Message() {
-    }
-    
-    public Message(long id, String messageContent, Date created, String username, String recipient) {
-        super();
-        this.id = id;
-        this.messageContent = messageContent;
-        this.created = created;
-        this.username = username;
-        this.recipient = recipient;
-    }
+import jakarta.json.bind.annotation.JsonbTransient;
+import jakarta.json.bind.annotation.JsonbDateFormat;
+
+@Entity
+@Table(name = "message")
+public class Message {
+
+	@JsonbProperty("id")
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+
+	@JsonbProperty("messageContent")
+	@Column(name = "messageContent")
```

Comment:
some column names are snake case and some are camel case ? can you maintain consistency ?

Reply:
table column names must always be snake case